test/data/pipeline/serialize.json.in

```diff
@@ -0,0 +1,14 @@
detic System 1984 (Transit)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1166\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G730)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1152\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G873)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1153\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G1150)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1154\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G1674)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1155\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G1762)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1156\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G2139)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1309\n            }\n          },\n          {\n            \"name\": \"World Geodetic System 1984 (G2296)\",\n            \"id\": {\n              \"authority\": \"EPSG\",\n              \"code\": 1383\n            }\n          }\n        ],\n        \"ellipsoid\": {\n          \"name\": \"WGS 84\",\n          \"semi_major_axis\": 6378137,\n          \"inverse_flattening\": 298.257223563\n        },\n        \"accuracy\": \"2.0\",\n        \"id\": {\n          \"authority\": \"EPSG\",\n          \"code\": 6326\n        }\n      },\n      \"coordinate_system\": {\n        \"subtype\": \"ellipsoidal\",\n        \"axis\": [\n          {\n            \"name\": \"Geodetic latitude\",\n            \"abbreviation\": \"Lat\",\n            \"direction\": \"north\",\n            \"unit\": \"degree\"\n          },\n          {\n            \"name\": \"Geodetic longitude\",\n            \"abbreviation\": \"Lon\",\n            \"direction\": \"east\",\n            \"unit\": \"degree\"\n          }\n        ]\n      },\n      \"id\": {\n        \"authority\": \"EPSG\",\n        \"code\": 4326\n      }\n    },\n    {\n      \"type\": \"VerticalCRS\",\n      \"name\": \"EGM96 height\",\n      \"datum\": {\n        \"type\": \"VerticalReferenceFrame\",\n        \"name\": \"EGM96 geoid\"\n      },\n      \"coordinate_system\": {\n        \"subtype\": \"vertical\",\n        \"axis\": [\n          {\n            \"name\": \"Gravity-related height\",\n            \"abbreviation\": \"H\",\n            \"direction\": \"up\",\n            \"unit\": \"metre\"\n          }\n        ]\n      },\n      \"id\": {\n        \"authority\": \"EPSG\",\n        \"code\": 5773\n      }\n    }\n  ]\n}"
```

Comment:
It's possible this PROJJSON isn't going to be stable. You might go try to read specific fields in the JSON for testing and such instead of doing any string comparisons.